src/docs/release-notes/R21/R21.mdx

```diff
@@ -0,0 +1,47 @@
+**Important**: we will be renamed to _Brightlayer User Interface_ (BLUI) soon, but our rapport to you all will stay the same 🤗
+
+### Component Libraries
+
+-   Updated Angular library to build with Angular 12.
+-   Documented all the components with JSDocs. (This is the intellisense thing)
+-   Enhanced React Native component library to allow icons passed directly into components.
+-   All PX Blue React Native component now responds to device rotation properly.
+-   Angular and React Native's Channel Value component now has the option to show / hide the space between unit and value.
+-   React Native hero component now takes ChannelValueProps to control the channel value child element.
+
+### Component Sticker Sheet
+
+-   Updated Figma [Component Sticker Sheet](https://www.figma.com/community/file/852558784352181868) to include 50+ (TODO: WAITING FOR SECURE CONNECT) new device- and channel-value-related icons. You can also find all of them in our [iconography](/style/iconography) page.
+-   All icons (Material Design and PX Blue icons) are now searchable by their aliases in Figma.
+-   Added more system components, including: phone keyboard, phone keypad, status bar, browser window, cursor, system dialog box and time picker.
+-   Added table components.
+
+### Themes
+
+-   Polished, pixel-perfect Angular Material chips.
+-   Snapshot testing to detect breaking changes in themes.
+-   Enhanced showcase demo for Angular and React to display all PX Blue and Material's components in an organized manner.
+-   Themed React Native Paper components.
+-   React Native dark theme, dropped RN dark theme alt.
+
+### Documentation Site
+
+-   Updated icon embedding instructions for React Native users.
+-   Updated Angular and React design pattern code examples for: Account Menu, App Bar, Forms & Validations and Loading & Waiting.
```

Comment:
Should this go under a "Design Patterns" or "Code Examples" heading rather than documentation site?

Reply:
Theoretically, PX Blue's users access design patterns from Doc-it. Unless this release note is written for upper management, I would leave it here.